File: src/main/kotlin/com/github/tobi812/sprykerplugin/models/generator/ParentGeneratorInterface.java
package com.github.tobi812.sprykerplugin.models.generator;

import pav.sprykerFileCreator.model.definition.ClassDefinitionInterface;
import pav.sprykerFileCreator.model.renderer.dto.PhpClassInterface;

public interface ParentGeneratorInterface {

    PhpClassInterface getParentClass(PhpClassInterface phpClass, ClassDefinitionInterface classDefinition);

}

File: src/main/kotlin/com/github/tobi812/sprykerplugin/models/generator/ParentGeneratorInterface.kt
package com.github.tobi812.sprykerplugin.models.generator

import com.github.tobi812.sprykerplugin.models.definitions.ClassDefinitionInterface
import com.github.tobi812.sprykerplugin.models.renderer.dto.PhpClassInterface

interface ParentGeneratorInterface {
    fun getParentClass(
        phpClass: PhpClassInterface,
        classDefinition: ClassDefinitionInterface
    ): PhpClassInterface?
}